demo/src/screens/foundationScreens/DarkModeScreen.tsx

```diff
@@ -0,0 +1,50 @@
+import React, {Component} from 'react';
+import {StyleSheet} from 'react-native';
+import {View, Text} from 'react-native-ui-lib';
+
+class DarkModeScreen extends Component {
+  state = {};
+  render() {
+    return (
+      <View flex padding-page bg-screenBG>
+        <Text h1 textColor>
+          Dark Mode
+        </Text>
+        <Text marginT-s2 body textColor>
```

Comment:
Consider showing this only on iOS [simulator](https://stackoverflow.com/a/35529545/4759619).